apps/organizations/apps.py

```diff
@@ -32,6 +32,34 @@ def organization_invitation_created(sender, instance, created, **kwargs):
                 slack_notify(msg)
             except:
                 rollbar.report_exc_info()
+    if created == False:
```

Comment:
In Python, do not compare booleans using `==`. Compare using the `is` operator.

For more on this, see: https://stackoverflow.com/questions/132988/is-there-a-difference-between-and-is

Also, you can/should just handle this using the `else` clause.